src/patternfly/components/Dropdown/styles.scss

```diff
@@ -17,7 +17,7 @@
   --pf-c-dropdown__toggle--BorderRadius: var(--pf-global--BorderRadius--sm);
   --pf-c-dropdown__toggle--Color: var(--pf-global--Color--100);
   --pf-c-dropdown__toggle--hover--BorderWidth: var(--pf-global--BorderWidth--md);
-  --pf-c-dropdown__toggle--hover--BorderColor: var(--pf-global--active-color--100);
+  --pf-c-dropdown__toggle--hover--BorderColor: var(--pf-global--BorderColor--dark);
```

Comment:
Why use `--pf-global--BorderColor--dark` here when the non-hover border color is `--pf-global--BorderColor`? Those colors are the same value, so it doesn't matter.

Although could we just remove the `border-color` variable and css, and just modify the `border-width` on hover?

Reply:
I've updated, but we need to keep the variable b/c it's used further down for the plain kebab hover. Although this exposes another issue of the plain hover when expanded vs the pf-m-action hover when expanded. I assume they should be the same.